ci/generate.go

```diff
@@ -61,3 +66,33 @@ func (m *Generate) WebSdk() *dagger.Directory {
 		Directory("/work/client/web").
 		WithoutDirectory("node_modules")
 }
+
+func (m *Generate) Check(ctx context.Context) error {
+	app := goModuleCross("").
+		WithSource(m.Source).
+		Container().
+		WithEnvVariable("GOFLAGS", "-tags=musl")
+
+	result := app.
+		WithExec([]string{"go", "generate", "-x", "-tags=musl", "-ldflags", "linkmode=external", "./..."}).
```

Comment:
```suggestion
		WithExec([]string{"go", "generate", "-x", "./..."}).
```

I don't think these are necessary here.